implementations/rust/ockam/ockam_abac/Cargo.toml

```diff
@@ -30,15 +30,18 @@ std = [
   "tokio",
   "wast",
   "strum/std",
+  "serde/std",
 ]
 
 [dependencies]
+cfg-if = "1.0.0"
 either = { version = "1.10.0", default-features = false }
 minicbor = { version = "0.20.0", features = ["derive", "alloc"] }
 ockam_core = { version = "0.102.0", path = "../ockam_core", default-features = false }
 ockam_identity = { version = "0.104.0", path = "../ockam_identity", default-features = false }
 ockam_node = { version = "0.109.0", path = "../ockam_node", default-features = false }
 once_cell = { version = "1.19.0", default-features = false, features = ["alloc"] }
+serde = { version = "1.0.197", default-features = false, features = ["derive"] }
```

Comment:
specifying the major only avoids dependency conflicts.

Reply:
I didn't know that, thanks for the heads up!